package-lock.json

```diff
@@ -22128,6 +22161,17 @@
         "node": ">=16"
       }
     },
+    "packages/cdn/node_modules/@curvenote/common": {
+      "version": "0.2.13",
+      "resolved": "https://registry.npmjs.org/@curvenote/common/-/common-0.2.13.tgz",
+      "integrity": "sha512-6l5IMBMjdFvE8/2THdBdx3u4IxhIszYcRnq6mKT/yh7PxWzfpZzhSuynbnEKPP97JishMM7SpD71ELZMZ9tlfg==",
+      "license": "MIT",
+      "dependencies": {
+        "@curvenote/blocks": "^1.5.29",
+        "myst-common": "^1.7.4",
+        "myst-config": "^1.7.4"
+      }
+    },
```

Comment:
This is expected?

Reply:
i.e. it is nested?

Reply:
This nesting is now gone (though it's slightly buried since the `package-lock.json` diff is larger...).